tox.ini

```diff
@@ -1,24 +1,20 @@
 [tox]
-envlist = py{27,35}-django{111}, py{35,36}-django{20,21,22},quality,docs
+envlist = py35-django22,py38-django{22,30},quality,docs
 
 [testenv]
 setenv =
-    # This allows us to reference test_settings.py
+ # This allows us to reference test_settings.py
     PYTHONPATH = {toxinidir}
-
-deps =
-    django111: Django>=1.11,<2.0
-    django20: Django>=2.0,<2.1
-    django21: Django>=2.1,<2.2
+deps = 
     django22: Django>=2.2,<2.3
+    django30: Django>=3.0,<3.1
     -rtest_requirements.txt
-
-commands =
+commands = 
     python -Wd manage.py test django_sites_extensions --settings=test_settings --with-coverage --cover-package=django_sites_extensions
     coverage report
 
 [testenv:quality]
-commands =
+commands = 
```

Comment:
Can we fix the update script for `tox.ini` to not add these trailing spaces?  It doesn't match what we had before, and adds a fair bit of diff noise.

Reply:
sure, I was planning to update it not to add Django 3.0 for python 3.5